source/fundamentals/indexes.txt

```diff
@@ -0,0 +1,268 @@
+=======
+Indexes
+=======
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+Indexes are data structures that support the efficient execution of queries in MongoDB. They contain copies of parts of
```

Comment:
The use of the phrase `data structures` might confuse people, as typically it is used to refer to in-memory constructs like linked lists or array rather than persistent on-disk representations.  Perhaps instead steal some language from the intro on https://docs.mongodb.com/manual/indexes/?